models/behavior.js

```diff
@@ -0,0 +1,9 @@
+class Behavior {
```

Comment:
that class is named `Behaviors`. I have been thinking of making the codebase a little more modular and renaming that class appropriately anyway. I will talk regarding this to Jeff today. Please join if possible.